packages/test/test-end-to-end-tests/src/test/gc/gcSweepAttachmentBlobs.spec.ts

```diff
@@ -36,7 +36,7 @@ import {
  * removed from the summary, added to the GC deleted blob, and retrieving them should be prevented.
  */
 describeCompat("GC attachment blob sweep tests", "NoCompat", (getTestObjectProvider) => {
-	const sweepTimeoutMs = 200;
```

Comment:
Same comment as above - this should be called sweepTimeoutMs and the sweepGracePeriod should be added to it.